frontend/src/app/components/about/about.component.html

```diff
@@ -177,14 +55,10 @@ <h3 i18n="about.contributors">Project Contributors</h3>
   <div class="maintainers">
     <h3 i18n="about.maintainers">Project Maintainers</h3>
     <div class="wrapper">
-        <a href="https://twitter.com/softsimon_" target="_blank" title="softsimon">
-          <img class="image" src="/resources/profile/softsimon.jpg" />
+        <a href="https://github.com/Rucknium" target="_blank" title="Rucknium">
```

Comment:
Is it okay to put Github link instead of twitter?

Reply:
Yes that sounds good. Thank you.